src/simulator/cpu/nbodysim.c

```diff
@@ -1,4 +1,5 @@
 #include "nbodysim.h"
+#include "tsneforces.c"
```

Comment:
As a warning here: I did the same thing on the GPU side (including the .c file directly), but this can cause Make to not detect changes in that file. So sometimes it will not recompile your binary with those new changes when you run make, so you need to `make clean && make` to get a proper build. Have been meaning to go and fix this on the GPU side but haven't gotten around to it yet